delivery_package_fee/models/sale_order_line.py

```diff
@@ -7,3 +7,9 @@ class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
     package_fee_id = fields.Many2one(comodel_name="delivery.package.fee", default=False)
+    is_delivery_package_fee = fields.Boolean(
+        string="Is a Delivery Package Fee", default=False,
```

Comment:
could we not compute this from the presence / absence of package_fee_id, and not resort on a field that has to be added in the database?

Reply:
I thought about that but, if we update the package fees defined on the carrier (removing them for instance) we will get an empty `package_fee_id` field on current order lines. I wanted to avoid that but if you think it's overkill I can still update this.

Reply:
set the field ondelete = restrict and make sure we have an active field on that model ?

Reply:
done